docs/orchestrating-elastic-stack-applications/elasticsearch/virtual-memory.asciidoc

```diff
@@ -92,7 +92,8 @@ spec:
           command: ['/usr/local/bin/bash', '-e', '-c', 'echo 262144 > /proc/sys/vm/max_map_count']
       containers:
         - name: sleep
-          image: gcr.io/google-containers/pause-amd64:3.2
+          image: docker.io/bash:5.2.21
```

Comment:
can we use the same version as in the init container?

Reply:
👍 fixed